src/modules/configurator/services/API_configurations.ts

```diff
@@ -75,12 +75,13 @@ export const fetchAllUserConfigurations = async (id: string): Promise<UserCarCon
 
         const configurationsArray: UserCarConfiguration[] = response.docs.map(config => {
             const userId = config.id;
-            const {model, model_id, production_year, color, wheel_variant, interior_variant, creation_date, total_price}: DocumentData = config.data();
+            const {model, model_id, model_price, production_year, color, wheel_variant, interior_variant, creation_date, total_price}: DocumentData = config.data();
```

Comment:
Sometimes it's better not to desctructure the variable, since you are making a copy of those variables. You could just write it as you did in the next function 

```
const configData = config.data();

...modelPrice: configData.model_price;
```

Reply:
Done.